mmdet/models/dense_heads/dense_test_mixins.py

```diff
@@ -34,8 +34,12 @@ def simple_test_bboxes(self, feats, img_metas, rescale=False):
                 with shape (n,)
         """
         outs = self.forward(feats)
-        results_list = self.get_bboxes(
-            *outs, img_metas=img_metas, rescale=rescale)
+        if getattr(self, 'static', False):
+            results_list = self.get_bboxes_statically(
```

Comment:
If put ipu-related logic into a subclass IPUYOLOV3Head, you don't have to change the box test mixin.